ProcessMaker/Console/Commands/Install.php

```diff
@@ -193,6 +197,56 @@ public function handle()
         return true;
     }
 
+    /**
+     * Configure the username, email, password of the admin user
+     *
+     */
+    private function fetchUserInformation()
+    {
+        do {
+            $username = $this->anticipate(__('Enter your username'), ['admin'], 'admin');
+            $validator = $this->validateField('username', $username, ['required', 'alpha_dash', 'min:4', 'max:255']);
+        } while($validator->fails());
+        do {
+            $email = $this->ask(__('Enter your email'));
+            $validator = $this->validateField('email', $email, ['required', 'email']);
+        } while($validator->fails());
+        do {
+            $password = $this->secret(__('Enter your password'));
+            $validator = $this->validateField('password', $password, ['required', 'sometimes', 'min:6']);
+        } while($validator->fails());
+        do {
+            $confirmPassword = $this->secret(__('Confirm your password'));
+            $match = $password === $confirmPassword;
+            if (!$match) {
+                $this->error(__('Your password/confirm password fields do not match'));
+            }
+        } while(!$match);
+        putenv ("INSTALLER_ADMIN_USERNAME=$username");
```

Comment:
While I like this, it's a security issue. Remove the write to ENV and just save to DB